types/src/transaction/transaction.rs

```diff
@@ -47,6 +47,36 @@ pub struct AssetOutPoint {
     pub amount: u64,
 }
 
+#[derive(Debug, Clone, Eq, PartialEq, RlpDecodable, RlpEncodable, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct Order {
+    // Main order informations
+    pub asset_type_from: H256,
+    pub asset_type_to: H256,
+    pub asset_type_fee: H256,
+    pub asset_amount_from: u64,
+    pub asset_amount_to: u64,
+    pub asset_amount_fee: u64,
+    /// previous outputs that order is started
+    pub origin_outputs: Vec<AssetOutPoint>,
+    /// expiration time by second
+    pub expiration: u64,
+    pub lock_script_hash: H160,
+    pub parameters: Vec<Bytes>,
+}
+
+#[derive(Debug, Clone, Eq, PartialEq, RlpDecodable, RlpEncodable, Deserialize, Serialize)]
```

Comment:
Why does it derive Serialize and Deserialize?

Reply:
It should have because of `chain_getTransaction`. I think `Serialize` can be deleted.